.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish to npm
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    name: Publish package
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
```

Comment:
we shouldn't need this as we are only publishing to npm rather than github packages